operations/mimir-mixin/dashboards/queries.libsonnet

```diff
@@ -336,13 +336,12 @@ local filename = 'mimir-queries.json';
       )
       .addPanel(
         $.panel('Index-header lazy load gate latency') +
-        $.queryPanel(
+        $.latencyPanel('cortex_bucket_stores_gate_duration_seconds{gate="index_header"}', '{%s}' % $.jobMatcher($._config.job_names.store_gateway)) +
+        $.panelDescription(
+          'Index-header lazy load gate latency',
+          |||
+            The average time spent waiting for a turn to load an index header. This time is not included in "Index-header lazy load duration."
```

Comment:
Now this description is slightly out of date 😞 It's not only the average. Maybe we can just say "Time spent waiting..."